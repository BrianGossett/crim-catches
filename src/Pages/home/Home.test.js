import renderer from 'react-test-renderer';
import Home from '../home';

describe('My Page', () => {
    it('is Displayed correctly', () =>{
        const tree = renderer.create(<Home />).toJSON();
        expect(tree).toMatchSnapshot();
    });
})