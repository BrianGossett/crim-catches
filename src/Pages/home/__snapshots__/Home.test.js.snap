// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Page is Displayed correctly 1`] = `
<div
  className="homepage"
>
  <h1>
    Home
  </h1>
</div>
`;

exports[`My Page is created empty 1`] = `
<div
  className="homepage"
>
  <h1>
    Home
  </h1>
</div>
`;
